config/viv_config.h

```diff
@@ -128,6 +128,10 @@ struct viv_libinput_config the_libinput_configs[] = {
         .left_handed = 0,
         // Invert scroll direction
         .natural_scroll = 0,
+        // Disable while typing
+        .disable_with_typing = LIBINPUT_CONFIG_DWT_ENABLED,
+        // Method used for registering the type of click
```

Comment:
Similarly, could you mention `(touchpad only)` here too?

Reply:
done